### DISCUSSION TITLE
Nested errors object with array - Object is possibly 'undefined'

### DISCUSSION THREAD

**Discussion by c0d3x** [react-hook-form/react-hook-form#3188] [Q&A]
How can I handle the error types when my errors are nested in an array, it says:
`Object is possibly 'undefined'`

![image](https://user-images.githubusercontent.com/661911/96253108-4cface80-0fb3-11eb-891d-6c999c15f8ab.png)

**Comment by bluebill1049** [comment]
can you share your type definition?

**Comment by p-viral** [comment]
I know I am late, but this is JS/TS error.

You can use `errors?.dabt?.[index]?.amount?.message ` to fix the TS error.

Cheers.

**Reply by SpurgeonPrakash** [reply]
but doing so giving linting errors saying errors  is possibly undefined when i build the next js app
